lib/Runtime/Library/JavascriptArray.h

```diff
@@ -6,6 +6,11 @@
 #pragma once
 
 #define ARRAY_CROSSOVER_FOR_VALIDATE 0
+#define ArrayAllocationBucket(ArrayType, index) ArrayType::allocationBuckets[index][0]
```

Comment:
How about using Getxxx and Setxxx  macros - instead of this way ?

---
In reply to: [82290881](https://github.com/Microsoft/ChakraCore/pull/1363#discussion_r82290881) [](ancestors = 82290881)